.github/CHANGELOG.md

```diff
@@ -203,4 +203,4 @@
 1. [ECAM] Added cockpit door video - @Benjozork (Benjamin Dupont)
 1. [MISC] Standby Instrument stays ON if emergency power should be available, bug fixes - @2hwk (2Cas#1022 on discord)
 1. [CDU] Full +/- button functionality - @lhoenig (Lukas Hoenig)
-1. [DCDU] Fixed MSG- and MSG+ button labels - @tyler58546 (tyler58546)
\ No newline at end of file
+1. [DCDU] Fixed MSG- and MSG+ button labels - @tyler58546 (tyler58546)
```

Comment:
You're changing the last line here. Please make sure your IDE does not mess with files unrelated to your commit.

Reply:
Idk why github(or my ide) always make a new line on last :(. Anyway, i remove that line